### PR TITLE
feat: Update OTEL image version to 0.94.0

### PR DESCRIPTION
**Pull request by hisarbalik**
## Description

Changes proposed in this pull request (what was done and why):

- Update OTEL image version to 0.94.0
- Add TracePipeline load test result with OTEL version 0.94.0
- Add MetricPipeline load test result with OTEL version 0.94.0

Changes refer to particular issues, PRs or documents:

- https://github.com/kyma-project/third-party-images/pull/418

## Traceability
- [x] The PR is linked to a GitHub issue.
- [ ] New features have a milestone set.
- [ ] New features have defined acceptance criteria in a corresponding GitHub Issue, and all criteria are satisfied with this PR.
- [x] The corresponding GitHub issue has a respective `area` and `kind` label.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.
- [x] Adjusted the documentation if the change is user-facing.
- [ ] The feature is unit-tested
- [ ] The feature is e2e-tested

### CODE REVIEW

**Review by NHingerl — APPROVED** [review]

File: docs/contributor/telemetry-load-test/assets/log-fluentbit-test-setup.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
       containers:
         - args:
             - --config=/etc/collector/config.yaml
-          image: europe-docker.pkg.dev/kyma-project/prod/tpi/otel-collector:0.93.0-fdc9e6c4
```
Comment (chrkl) [inline docs/contributor/telemetry-load-test/assets/log-fluentbit-test-setup.yaml:87]: Was this intentional?

Comment (hisarbalik) [reply docs/contributor/telemetry-load-test/assets/log-fluentbit-test-setup.yaml:87]: yes, we will use official OTEL image release for mock backend, logging mock backend had different image than trace and metric mock backend

Comment (chrkl) [reply docs/contributor/telemetry-load-test/assets/log-fluentbit-test-setup.yaml:87]: But shouldn't we also use 0.94.0?

Comment (chrkl) [reply docs/contributor/telemetry-load-test/assets/log-fluentbit-test-setup.yaml:87]: And what is missing in our image? We add the fluentforwardreceiver [here](https://github.com/kyma-project/third-party-images/blob/main/otel-collector/builder-config.yaml#L16)

**Review by chrkl — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**